us/en/skymiles/styles/styles.css

```diff
@@ -19,6 +19,7 @@
   --color-delta-gray-middle: #d2d3d5;
   --color-delta-gray-middle2: #e6e6e6;
   --color-delta-gray-light: #f6f6f7;
+  --color-delta-gray-border: #dedfe1;
```

Comment:
I would just re-use `--color-delta-gray-middle`. It's close enough color-wise that nobody would really notice and it simplifies the color palette.